libraries/exoplayer/src/main/java/androidx/media3/exoplayer/drm/FrameworkMediaDrm.java

```diff
@@ -380,6 +382,21 @@ public FrameworkCryptoConfig createCryptoConfig(byte[] sessionId) throws MediaCr
     return C.CRYPTO_TYPE_FRAMEWORK;
   }
 
+  @Override
+  public void removeOfflineLicense(byte[] keySetId) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
I don't think it's really acceptable to make this method a no-op on pre-Q devices without making that much clearer in the interface.

See for example the docs on `setOnKeyStatusChangeListener` and the implementation that throws an exception on unsupported API levels.

Reply:
Thanks for reviewing. Updated.